### ISSUE TITLE
Add support for MySQL Cluster

### ISSUE THREAD

**Issue by datacharmer** [datacharmer/dbdeployer#20]
This addition should work, in principle, in the same way that group replication does.

The implementation will require:

* identifying which tarballs support this feature out of the box (i.e. no separate downloads);
* pre-define the my.cnf options needed for the installation
* deploy a multiple sandbox (similar to what we do for multi-source replication
* run a configuration script.

``defaults.go`` has already potential support for MySQL Cluster ports handling (commented out).

**Comment by dveeden** [comment]
Hi,

I assume this is about NDB cluster and not about InnoDB Cluster?
```
// NdbBasePort                    int    `json:"ndb-base-port"
```

Note that MySQL Cluster 8.0 DMR is now on the same version scheme as "regular" MySQL.

For NDB a single sandbox would make sense, just with multiple data nodes.

With MySQL::Sandbox I only added `ndbcluster` to my.sandbox.cnf. This is with 7.x.
Outside of MySQL::Sandbox I have a config.ini and start ndb_mgmd and multiple ndbmtd processs.

**Comment by datacharmer** [comment]
Implemented with PR #71
